### PR TITLE
20 typos minor fixes

### PR DESCRIPTION
**Pull request by thecowsays**
- Fixed some typos in the CSS and updated anywhere the fix was referenced
- Renamed some vars() in CSS to match their intended use (removed specific color names for global color schemes), updated any references
- Fixed typos in HTML and CSS _id_ names & updated any references
- Adjusted whitespace in files
- Updated comments for documentation

### CODE REVIEW

File: browse.html [hunk 1]
```diff
@@ -36,72 +38,72 @@
...
+                        </section>
+                        <section>
+                            <input type="checkbox" id="gin" name="GIN">
+                            <label for="gin">GIN</label>
+                        </section>
+                        <section>
+                            <input type="checkbox" id="rum" name="RUM">
+                            <label for="rum">RUM</label>
+                        </section>
+                    </div>
```
Comment (thecowsays) [inline browse.html:60]: Notice in the changed code, the div closing immediately follows section closing tags...

File: css/style.css [hunk 2]
```diff
@@ -1,365 +1,392 @@
-/* Color Scheme. Please use var() to set colours. This will make site-wide changes later much easier */
-:root{
-    --primary-grey:#222831;
```
Comment (thecowsays) [inline css/style.css:3]: By using color names, it defeats the purpose of having a _global variable_. What if we wanted to change from grey to blue? Would we change this and all references to blue?

File: css/style.css [hunk 3]
```diff
@@ -1,365 +1,392 @@
...
 
-
-
-/* General font styling (check figma file for notes on how to use!) */
-h1,h2,h3{
+h1,
+h2,
+h3 {
     font-family: 'Bungee Inline', cursive;
-    color: var(--accent-yellow);
```
Comment (thecowsays) [inline css/style.css:25]: Keeping with the "global" theme, this color name was removed from the actual var, like the others. All referenced uses were updated so the name changes wouldn't break.

**Review by MidnightFlamingo97 — APPROVED** [review]
Approving merge. Issue surrounding the </div> tags is a non-issue. Code works fine and is good to push to main. Also good practice point for others to re-base their feature branches with the updated main branch to avoid future conflicts

File: browse.html [hunk 4]
```diff
@@ -36,72 +38,72 @@
...
-                        <label for="whiskey">WHISKEY</label>
-                    </section>
-                    <section>
-                        <input type="checkbox" id="gin" name="GIN">
-                        <label for="gin">GIN</label>
-                    </section>
-                    <section>
-                        <input type="checkbox" id="rum" name="RUM">
-                        <label for="rum">RUM</label>
-                    </section>
```
[the remark this answers is not in the document]
Comment (MidnightFlamingo97) [reply browse.html:57]: Non-issue. </div> is present below

File: browse.html (hunk 1 above)
Comment (MidnightFlamingo97) [reply browse.html:60]: original </div> on line 58 / 61

File: css/style.css [hunk 5]
```diff
@@ -1,365 +1,392 @@
...
- .select-ingredients-container section{
-     padding: 10px 0;
-     font-size: 18px;
- }
- #vodka{
-     border: none;
-     color: var(--accent-yellow);
-     background: transparent;
- }
-
```
[the remark this answers is not in the document]
Comment (MidnightFlamingo97) [reply css/style.css:240]: No problem

File: css/style.css (hunk 3 above)
Comment (MidnightFlamingo97) [reply css/style.css:25]: Cool

File: css/style.css (hunk 2 above)
Comment (MidnightFlamingo97) [reply css/style.css:3]: Yes good point, accepted change

File: css/style.css [hunk 6]
```diff
@@ -1,365 +1,392 @@
-/* Color Scheme. Please use var() to set colours. This will make site-wide changes later much easier */
-:root{
-    --primary-grey:#222831;
-    --seconday-grey:#393E46;
```
[the remark this answers is not in the document]
Comment (MidnightFlamingo97) [reply css/style.css:4]: Lol

**Pull request merged**